### DISCUSSION TITLE
can pages created using CreatePage API ignore case

### DISCUSSION THREAD

**Discussion by gowthamik14** [gatsbyjs/gatsby#36904] [Help]
I have a requirement where User want to navigate to URL  /field/stat , but the URL is generated using CreatePage in gatsby.node.js has Feild/STAT. 
Is there anyway we can ignore the case on generated pages. Is it possible. Appreaciate if there is any work around.

**Comment by LekoArts** [comment]
You have to set up server redirects

**Reply by gowthamik14** [reply]
could you please provide an example

**Comment by tsdexter** [comment]
Do you have a code example @gowthamik14? URLs are case sensitive, so you would need to add a redirect in `gatsby-node.js` like: 

```js
createRedirect({
  fromPath: `/Field/STAT`,
  toPath: `/field/stat`,
})
```

If you're in `gatsby-node.js` anyway, it would make more sense to just add `.toLowerCase()` in your `createPage` but it's hard to tell what the best solution is without seeing your code or an example.
